x-pack/plugins/enterprise_search/server/lib/ml/fetch_ml_models.ts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MlTrainedModelConfig, MlTrainedModelStats } from '@elastic/elasticsearch/lib/api/types';
+import { i18n } from '@kbn/i18n';
+import { MlTrainedModels } from '@kbn/ml-plugin/server';
+import { SUPPORTED_PYTORCH_TASKS } from '@kbn/ml-trained-models-utils';
+
+import { MlModelDeploymentState, MlModel } from '../../../common/types/ml';
+
+export const ELSER_MODEL_ID = '.elser_model_2';
```

Comment:
Are these model names defined elsewhere that we can bring in?

Reply:
There is an API to [get the ELSER config](https://github.com/elastic/kibana/blob/1823d94240abadd456c145329ea83410ec485e26/x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts#L76), including the model name, that's the proper way to do it AFAIK. But I don't see the equivalent for E5.